tests/covscripts/parse_attributes.py

```diff
@@ -0,0 +1,12 @@
+import json
+
+coverage_file = open('../MITunaX/tests/covscripts/buffer/coverage.json')
```

Comment:
get current path and formulate the path to coverage.json that way. Check out some of our tests/<>.py scripts